<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettingsDialog</class>
 <widget class="QDialog" name="AppearanceSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>505</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="minSizeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum icon size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minSize">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimum">
    <number>16</number>
   </property>
   <property name="maximum">
    <number>64</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="maxSizeLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum icon size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxSize">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimum">
    <number>32</number>
   </property>
   <property name="maximum">
    <number>192</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundAlphaLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Panel opacity</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="backgroundAlpha">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundColorLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Background color</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showBorder">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Show panel border</string>
   </property>
  </widget>
  <widget class="QLabel" name="borderColorLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Border color</string>
   </property>
  </widget>
  <widget class="QLabel" name="tooltipFontSizeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tooltip font size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tooltipFontSize">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>28</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AppearanceSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AppearanceSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
